internal/vendor/wiz.go

```diff
@@ -423,6 +450,20 @@ type KubernetesCluster struct {
 	IsConnectedUsingBroker bool         `json:"isConnectedUsingBroker"`
 }
 
+// ProjectKubernetesClusterLink struct
+type ProjectKubernetesClusterLink struct {
+	ID                string                   `json:"id"`
+	KubernetesCluster ProjectKubernetesCluster `json:"kubernetesCluster"`
+	Environment       string                   `json:"environment"`
+	Namespaces        []string                 `json:"namespaces"`
+	Shared            bool                     `json:"shared"`
+}
+
+// ProjectKubernetesCluster struct
+type ProjectKubernetesCluster struct {
+	ID string `json:"id"`
+}
+
```

Comment:
ProjectKubernetesCluster is not defined in the Wiz GraphQL schema.  Did you instead mean KubernetesCluster or ProjectKubernetesClusterLink or ProjectKubernetesClusterLinkInput?